pages/index.vue

```diff
@@ -0,0 +1,519 @@
+                creando <br>
+                un circuito elitario di club <br>
+                diffusi <span class="inline-flex items-center gap-1 bg-secondary text-app-bg  px-2 my-1">
+                    globalmente
+                    <icon-globe />
+                </span> attraverso <br>
+                la tecnologia web3.
+            </p>
+            <span
+                class=" text-app-bg bg-[#D3E2E8] px-2 md:px-4 py-2 text-xl md:text-3xl absolute  -top-20 left-6 md:-left-2 rotate-[-5.45deg]">IL
+                PROGETTO</span>
+        </div>
+    </section>
+
+    <!-- marque section -->
+    <section class="mt-40 lg:mt-[131px] pt-11 pb-16 border border-white border-l-0 border-r-0 border-opacity-25">
+        <vue3-marquee :clone="true" :duration="20" :direction="'reverse'" :pause-on-hover="true"
+            @onOverflowCleared="alert()" class=" overflow-y-hidden">
+            <div class="text-white font-medium text-4xl md:text-6xl lg:text-[86px] flex items-center gap-5 italic hover:text-[#4D696D] hover:not-italic mr-10 transition duration-200 ease-linear"
+                v-for="text in marqueText" :key="text">
+                {{ text }}
+                <icon-arrow-right-large class="w-12 h-12 md:w-24 md:h-24 lg:w-[91px] lg:h-[84px]" />
+            </div>
+        </vue3-marquee>
+    </section>
+
+    <!-- partner section -->
+    <section class="mt-[156px]">
+        <div class="container ">
+            <div class="bg-[#1A3E43] relative max-w-[1254px] mx-auto py-8" data-aos="fade-up">
+                <div
+                    class="text-4xl lg:text-3xl  xl:text-5xl lg:!leading-[56px] pt-[92px] pb-[61px] pl-4 md:pl-[53px] text-[#99A9AB] relative">
+                    <span
+                        class=" rotate-[-3.82deg] bg-secondary absolute -top-5 text-app-bg text-2xl lg:text-3xl px-4 py-2">PERCHÈ
+                        DIVENTARE
+                        PARTNER</span>
+                    Per entrare <b><em class="text-white">a far parte</em></b> <br />
+                    di un <b><em class="text-white">network</em></b> selezionato <br />
+                    di aziende legate al <b><em class="text-white">mondo</em></b> <br />
+                    <b><em class="text-white">del padel</em> e connettersi</b> <br />
+                    con una <b><em class="text-white">community globale</em></b> <br />
+                    di <b><em class="text-white">appassionati, sportivi</em></b>, <br />
+                    <b><em class=" text-white">aziende del settore</em></b> e investitori <br />
+                    <b><em class="text-white">interessati alle innovazioni</em></b> <br>
+                    web3. 
+                </div>
+                <img src="/partner.png" alt=""
+                    class="absolute -right-10 top-1/2 transform -translate-y-1/2 hidden lg:inline " />
+            </div>
+
+        </div>
+    </section>
+
+    <!-- new era section -->
+    <section class="mt-24 md:mt-[203px]">
```

Comment:
instead use `mt-52` which is `208px`

Reply:
i don't continue but try to uplade all the inline styles with the most near class of tailwindcss